packages/protocol/contracts/governance/Governance.sol

```diff
@@ -649,7 +672,71 @@ contract Governance is IGovernance, Ownable, Initializable, UsingLockedGold, Ree
   }
 
   /**
-   * @notice Withdraws refunded Celo Gold commitments.
+   * @notice Whitelists the hash of a hotfix.
+   * @param txHash The abi encoded keccak256 hash of the hotfix transaction to be whitelisted.
+   * @param validatorIndex The index in the validator set of the whitelisting validator.
+   */
+  function whitelistHotfix(bytes32 txHash, uint256 validatorIndex) external nonReentrant {
+    if (msg.sender == approver) {
```

Comment:
How do we want to deal with changing `validator` keys. For example, if I lock gold up with key A, authorize key B as my validator key, whitelist a hotfix with key B, and then authorize key C as my validator key, is the hotfix still considered whitelisted by me?